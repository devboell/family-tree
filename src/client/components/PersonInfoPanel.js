import styled from 'styled-components'

const PersonInfoPanel = styled.div`
  border: 1px solid black;
  border-radius: 5px;
  width: ${props => props.width}px;
  height: ${props => props.height}px;
  > p {
      margin-top: 5px;
      text-align: center;
      font-size: 14px;
      color: blue;
    }
`

export default PersonInfoPanel
