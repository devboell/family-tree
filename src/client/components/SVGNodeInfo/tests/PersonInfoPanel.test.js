import React from 'react'
import 'jest-styled-components'

import PersonInfoPanel from '../PersonInfoPanel'

const props = {
  width: 100,
  height: 100,
}

it('PersonInfoPanel, styled component snapshot', () => {
  const wrapper = shallow(<PersonInfoPanel {...props} />)
  expect(wrapper).toMatchStyledComponentsSnapshot()
})
