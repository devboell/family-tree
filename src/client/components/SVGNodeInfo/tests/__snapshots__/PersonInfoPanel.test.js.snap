// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonInfoPanel, styled component snapshot 1`] = `
.ixIxkP {
  border: 1px solid black;
  border-radius: 5px;
  width: 100px;
  height: 100px;
}

.ixIxkP > p {
  margin-top: 5px;
  text-align: center;
  font-size: 14px;
  color: blue;
}

<div
  className="sc-bdVaJa ixIxkP"
  height={100}
  width={100}
/>
`;
